Dockerfile

```diff
@@ -15,23 +15,24 @@
 FROM alpine:3.11 as chart-get
 
 RUN apk add wget tar
-RUN wget https://charts.jetstack.io/charts/cert-manager-v0.15.0-alpha.0.tgz
+RUN wget https://charts.jetstack.io/charts/cert-manager-v0.15.0-alpha.1.tgz
 # TODO: add some kind of verification
-RUN tar xzf cert-manager-v0.15.0-alpha.0.tgz
+RUN tar xzf cert-manager-v0.15.0-alpha.1.tgz
+
+FROM quay.io/jetstack/cert-manager-controller:v0.15.0-alpha.1 as cm-image
 
 FROM quay.io/operator-framework/helm-operator:v0.15.1
 
 ### Required OpenShift Labels
 LABEL name="cert-manager Operator" \
       vendor="Jetstack" \
-      version="v0.15.0-alpha.0" \
+      version="v0.15.0-alpha.1" \
       release="1" \
       summary="This is the cert-manager operator." \
       description="This operator will deploy cert-manager to the cluster."
 
 # Required Licenses
-ADD https://raw.githubusercontent.com/jetstack/cert-manager/v0.15.0-alpha.0/LICENSE /licenses/LICENSE
-ADD https://raw.githubusercontent.com/jetstack/cert-manager/v0.15.0-alpha.0/LICENSES /licenses/LICENSES
+COPY --from=cm-image /licenses /licenses
```

Comment:
Hm, a thought.. should the operator contain licenses for the controller component, or should it contain LICENSES for the actual operator component?

Reply:
very good point! we should also copy the ones from the operator over

Reply:
I will add all licences there, better to have too much than too less